packages/loader/container-loader/src/container.ts

```diff
@@ -1430,7 +1426,20 @@ export class Container extends EventEmitterWithErrorHandling<IContainerEvents> i
             this.emit("readonly", readonly);
         });
 
-        return deltaManager;
+        // Add "active" property.
+        // The meaning of it is Container specific, as we allow container runtime to send ops
+        // only when we are connected as "write" and are up to date, i.e. observed our own join op.
+        // The later info does not exists at DeltaManager layer, it's Container specific logic.
+        Object.defineProperties(deltaManager, {
```

Comment:
i think i'd rather just pass in a callback, rather than do this

Reply:
I thought about callback, but my ideal solution would be not not even expose this concept in DeltaManager.
I guess there is no good choice here :(